includes/Story_Post_Type.php

```diff
@@ -610,6 +613,106 @@ public function filter_use_block_editor_for_post_type( $use_block_editor, $post_
 		return $use_block_editor;
 	}
 
+	/**
+	 * Returns list of custom meta boxes per location.
+	 *
+	 * Used to disable empty meta boxes in the editor.
+	 *
+	 * @see the_block_editor_meta_boxes()
+	 *
+	 * @since 1.2.0
+	 *
+	 * @return array List of meta boxes per location.
+	 */
+	public function get_meta_boxes_per_location() {
+		global $wp_meta_boxes;
+
+		$screen = get_current_screen();
+
+		if ( ! $screen instanceof WP_Screen ) {
+			return [];
+		}
+
+		/* This filter is documented in wp-admin/includes/post.php */
+		$_wp_meta_boxes = apply_filters( 'filter_block_editor_meta_boxes', $wp_meta_boxes );
+
+		$priorities = [ 'high', 'sorted', 'core', 'default', 'low' ];
+
+		$meta_boxes_per_location = [];
+		foreach ( [ 'normal', 'advanced' ] as $context ) {
+			$meta_boxes_per_location[ $context ] = [];
+
+			if ( ! isset( $wp_meta_boxes[ $screen->id ][ $context ] ) ) {
+				continue;
+			}
+
+			foreach ( $priorities as $priority ) {
+				if ( ! isset( $_wp_meta_boxes[ $screen->id ][ $context ][ $priority ] ) ) {
+					continue;
+				}
+
+				$meta_boxes = (array) $_wp_meta_boxes[ $screen->id ][ $context ][ $priority ];
+				foreach ( $meta_boxes as $meta_box ) {
+					if ( false === $meta_box || ! $meta_box['title'] ) {
+						continue;
+					}
+
+					$meta_boxes_per_location[ $context ][] = [
+						'id'    => $meta_box['id'],
+						'title' => $meta_box['title'],
+					];
+				}
+			}
+		}
+
+		return $meta_boxes_per_location;
+	}
+
+	/**
+	 * Removes all meta boxes with '__back_compat_meta_box' set to 'true'.
+	 *
+	 * This removes things like the post author meta box, as this feature is
+	 * already included in the editor.
+	 *
+	 * Mimics what do_meta_boxes() does for the block editor.
+	 *
+	 * @see do_meta_boxes()
+	 *
+	 * @since 1.2.0
+	 *
+	 * @return void
+	 */
+	public function remove_meta_boxes() {
+		global $wp_meta_boxes;
+
+		$screen = get_current_screen();
+
+		if ( ! $screen instanceof WP_Screen ) {
+			return;
+		}
+
+		// We don't currently support the 'Custom Fields' meta box.
+		remove_meta_box( 'postcustom', $screen, $screen->id );
+
+		foreach ( [ 'normal', 'advanced' ] as $context ) {
+			if ( ! isset( $wp_meta_boxes[ $screen->id ][ $context ] ) ) {
+				continue;
+			}
+
+			foreach ( [ 'high', 'sorted', 'core', 'default', 'low' ] as $priority ) {
+				foreach ( (array) $wp_meta_boxes[ $screen->id ][ $context ][ $priority ] as $meta_box ) {
```

Comment:
```suggestion
                                 if ( !isset( $wp_meta_boxes[ $screen->id ][ $context ][ $priority ] )  ){
                                       continue;
                                 }
                                  
				foreach ( (array) $wp_meta_boxes[ $screen->id ][ $context ][ $priority ] as $meta_box ) {
```

This fixed a notice warning I got locally.